File: URI/1 - INICIANTE/Python/1178 - PreenchimentoDeVetor3.py
vetor = []

num = float(input())
vetor.append(num)
metade = num/2
for y in range(1, 100):
    vetor.append(metade)
    metade /= 2
for y in range(0,100):
    print('N[{}] = {:.4f}'.format(y, vetor[y]))

File: URI/1 - INICIANTE/C/1178 - PreenchimentoDeVetor3.c
#include <stdio.h>

int main(){
	double vetor[100], metade, num;
	int x;
	scanf(" %lf", &num);
	vetor[0] = num;
	metade = num/2;
	for(x = 1; x < 100; x++){
		vetor[x] = metade;
		metade /= 2;
	}
	for(x = 0; x < 100; x++){
		printf("N[%d] = %.4f\n", x, vetor[x]);
	}
	
	return 0;
}
